manifests/plugin/dynflow.pp

```diff
@@ -70,5 +73,10 @@
       ensure => running,
       enable => true,
     }
+    ~> systemd::service_limits { 'smart_proxy_dynflow_core.service':
```

Comment:
This is the wrong way around. Right now the service notifies the limits rather than the limits the service.